gateway-service/src/main/java/org/zowe/apiml/gateway/ws/WebSocketProxyServerHandler.java

```diff
@@ -223,25 +232,74 @@ private void close(WebSocketSession session, CloseStatus status) {
         }
     }
 
+    @Recover
+    void closeServerSession(ServerNotYetAvailableException e, WebSocketSession webSocketSession, WebSocketMessage<?> webSocketMessage) {
```

Comment:
I can't find where this ServerNotYetAvailableException exception is being used

Reply:
Yes, it's not used, it's just for the method annotated with `@Recover` to get the original exception injected. In this case (for now) I know that the only moment this is called is when it exhausted the retries for this particular exception

Reply:
ok got it, thanks!